android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -1409,6 +1409,7 @@ sealed class SearchAction : BrowserAction() {
         val regionSearchEngines: List<SearchEngine>,
         val customSearchEngines: List<SearchEngine>,
         val hiddenSearchEngines: List<SearchEngine>,
+        val disabledSearchEngineShortcutsIds: List<String>,
```

Comment:
```suggestion
        val disabledSearchEngineIds: List<String>,
```

Drive-by: you missed one more reference to update. 🙂